kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/serialization/TransientInjectableObjectOutputStream.java

```diff
@@ -279,6 +295,21 @@ protected Object replaceObject(Object obj) {
         return obj;
     }
 
+    private void handleUnserializableComponentWrapper(
+            UnserializableComponentWrapper<?, ?> wrapper) {
+        if (!wrapper.isAttached()) {
```

Comment:
I think this method could be invoked twice, once for the `UnserializableComponentWrapper` object and then for its `Element`. We can probably add a check and skip the logic if the `wrapper` is already present in the `unserializableComponents` set.

Reply:
Yes, it was invoked twice. Added the check if the wrapper is already present in the `unserializableComponents`.